OpenRA.Mods.Common/Traits/Render/WithRotor.cs

```diff
@@ -28,9 +28,6 @@ public class WithRotorInfo : ITraitInfo, IRenderActorPreviewSpritesInfo, Require
 		[Desc("Position relative to body")]
 		public readonly WVec Offset = WVec.Zero;
 
-		[Desc("Change this when using this trait multiple times on the same actor.")]
-		public readonly string Id = "rotor";
```

Comment:
We should clean the YAML in such cases. If we start leaving stuff like this soon half of our YAML files will be unused junk values.